src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -319,21 +713,40 @@ public void binaryContentContentTypeAndType(final String uri) {
                         "type of the external content, if provided. Any Content-Type header in the request should be " +
                         "ignored.",
                 "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        final Headers headers = new Headers(
```

Comment:
I believe this element of the specification is referring to `Content-Type` in the creation request. Therefore, I would suggest adding a `Content-Type` header to the `headers` list.

Reply:
Okay, I definitely misinterpreted that, will fix